operator/v1/types_baremetal.go

```diff
@@ -0,0 +1,68 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+
+// Metal3ProvisioningController contains configuration used by the Provisioning service (Ironic)
```

Comment:
Explain what the different instances of this resource do.  Are they required to match one to one to ingress controllers?  If they don't, what does it do?

Reply:
Currently we expect to have exactly one of these resources to provide data for the metal3 pod started by the MAO.

Reply:
> Explain what the different instances of this resource do. Are they required to match one to one to ingress controllers? If they don't, what does it do?

As @hardys mentioned, we should have only one instance of this CR. No connection with the IngressController.